src/types/server-transations.ts

```diff
@@ -0,0 +1,243 @@
+import { AssetType, ScamInfoType } from './account';
+import { NftEnumType } from './tokens';
+
+export interface ScResultType {
+  callType: string;
+  gasLimit: number;
+  gasPrice: number;
+  nonce: number;
+  prevTxHash: string;
+  hash: string;
+  originalTxHash: string;
+  receiver: string;
+  sender: string;
+  timestamp: number;
+  value: string;
+  data?: string;
+  returnMessage?: string;
+}
+
+export interface TransactionTokensType {
+  esdts: string[];
+  nfts: string[];
+}
+
+export enum TxActionsEnum {
+  // esdtNft category
+  transfer = 'transfer',
+  // stake category
+  delegate = 'delegate',
+  stake = 'stake',
+  unDelegate = 'unDelegate',
+  stakeClaimRewards = 'claimRewards',
```

Comment:
I think we should split this into multiple enums and make a union, this enum is too big